src/custom/components/CurrencyInputPanel/CurrencyInputPanelMod.tsx

```diff
@@ -25,12 +26,6 @@ import { formatSmart } from 'utils/format'
 import { SHORT_PRECISION } from 'constants/index'
 import { FeeInformationTooltipWrapper } from 'components/swap/FeeInformationTooltip'
 
-export const CurrencySearchModal = styled(CurrencySearchModalUni)`
```

Comment:
Didn't seem part of the original UNI file. Moved to the mod index.

Reply:
👍